code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -50,7 +50,7 @@ Difficulty: Hard
 	armour_penetration = 50
```

Comment:
this file has a lot of sleeps w/o sanity (do we just call those nightmares?)

Reply:
4d was doing #define for sleeps that check for qdeleted or dead at the end of the sleep should probably replace every sleep in mobs with that type of stuff not gonna lie